fhservices.cfg.j2

```diff
@@ -57,26 +57,26 @@ define host {
        host_name {{ rhmap_router_dns }}
 }
 
-{% for fhservice in fh_services_ping %}
+{% for fhservice in fh_services if 'ping' in fhservice.get("checks") %}
```

Comment:
There is no obvious sensible default for 'name' or 'checks' in a service config. We could have that default to an empty array, but that would only mask a potential issues with the config being passed in. Currently if this fails it will create an invalid config and the container will fail to start which seems preferable. wdyt?